### PR TITLE
update release notes for 0.4.14

### PR DESCRIPTION
**Pull request by alisterburt**
# Description
minor update to release notes for new PRs since #4013

### CODE REVIEW

File: docs/release/release_0_4_14.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is a small release containing some new features and fixing a few issues tha
 
 This release adds the ability to convert a shapes layer to a labels layer from a context menu available on right clicking the layer list entry (#3978). Our vectors layer has gained the 
 ability to render vectors in 2D across multiple slices (#2902) and points in the points
-layer can now be hidden individually (#3625).
+layer can now be hidden individually (#3625). Support has also been added for (opt-in only) error reporting using `napari-error-reporting`.
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:13]:
```suggestion
layer can now be hidden individually (#3625).
```

File: docs/release/release_0_4_14.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ Complete list of changes below:
 - Hide points individually (#3625)
 
 ## New Features
+- Support error reporting via napari-error monitor (#4040)
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
```

File: docs/release/release_0_4_14.md (hunk 1 above)
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
layer can now be hidden individually (#3625).
```

**Review by sofroniewn — APPROVED** [review]
LGTM!

**Comment by brisvag** [comment]
Number of reviewers is outdated!

**Review by alisterburt — COMMENTED** [review]
thanks both!!

**Pull request merged**